document/cloud-client/parse_table_beta.py

```diff
@@ -0,0 +1,105 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START documentai_parse_table]
+from google.cloud import documentai_v1beta2 as documentai
+
+
+def parse_table(project_id='YOUR_PROJECT_ID',
+                input_uri='gs://cloud-samples-data/documentai/invoice.pdf'):
+    """Parse a form"""
+
+    client = documentai.DocumentUnderstandingServiceClient()
+
+    gcs_source = documentai.types.GcsSource(uri=input_uri)
+
+    # mime_type can be application/pdf, image/tiff,
+    # and image/gif, or application/json
+    input_config = documentai.types.InputConfig(
+        gcs_source=gcs_source, mime_type='application/pdf')
+
+    # Improve table parsing results by providing bounding boxes
+    # specifying where the box appears in the document (optional)
+    table_bound_hints = [
+        documentai.types.TableBoundHint(
+            page_number=1,
+            bounding_box=documentai.types.BoundingPoly(
+                # Define a polygon around tables to detect
+                # Each vertice coordinate must be a number between 0 and 1
+                normalized_vertices=[
+                    # Top left
+                    documentai.types.geometry.NormalizedVertex(
+                        x=0,
+                        y=0
+                    ),
+                    # Top right
+                    documentai.types.geometry.NormalizedVertex(
+                        x=1,
+                        y=0
+                    ),
+                    # Bottom right
+                    documentai.types.geometry.NormalizedVertex(
+                        x=1,
+                        y=1
+                    ),
+                    # Bottom left
+                    documentai.types.geometry.NormalizedVertex(
+                        x=0,
+                        y=1
+                    )
+                ]
+            )
+        )
+    ]
+
+    # Setting enabled=True enables form extraction
+    table_extraction_params = documentai.types.TableExtractionParams(
+        enabled=True, table_bound_hints=table_bound_hints)
+
+    # Location can be 'us' or 'eu'
+    parent = 'projects/{}/locations/us'.format(project_id)
+    request = documentai.types.ProcessDocumentRequest(
+        parent=parent,
+        input_config=input_config,
+        table_extraction_params=table_extraction_params)
+
+    document = client.process_document(request=request)
+
+    def _get_text(el):
+        """Convert text offset indexes into text snippets.
+        """
```

Comment:
nit: preference for helper method to match across files

Reply:
Acknowledged.

I'll add that in a fast-follow PR.